### PR TITLE
Add exponentional easing functions

### PR DESCRIPTION
**Pull request by typfel**

### CODE REVIEW

File: RBBAnimation/RBBEasingFunction.m [hunk 1]
```diff
@@ -85,3 +85,22 @@
...
+RBBEasingFunction const RBBEasingFunctionEaseInOutExpo = ^(CGFloat t) {
+    if (t == 0) {return 0.0;}
+    if (t == 1) {return 1.0;}
+    
+    if (t < 0.5) {
+        return POW(2, 10 * (2 * t - 1)) / 2;
+    } else {
+        return 1 - POW(2, -10 * (2 * t - 1)) / 2;
+    }
+};
```
Comment (robb) [inline RBBAnimation/RBBEasingFunction.m]: Missing new line at end of file.

File: RBBAnimation/RBBEasingFunction.m [hunk 2]
```diff
@@ -85,3 +85,22 @@
...
+    return t == 0 ? 0.0 : POW(2, 10 * (t - 1));
+};
+
+RBBEasingFunction const RBBEasingFunctionEaseOutExpo = ^(CGFloat t) {
+    return (t == 1.0) ? 1 : 1 - POW(2, - 10 * t);
+};
+
+RBBEasingFunction const RBBEasingFunctionEaseInOutExpo = ^(CGFloat t) {
+    if (t == 0) {return 0.0;}
+    if (t == 1) {return 1.0;}
```
Comment (robb) [inline RBBAnimation/RBBEasingFunction.m]: Can you make these either

```objc
if (t == 1.0) return 1.0
```

or

```objc
if (t == 1.0) {
    return 1.0
}
```
?

**Comment by robb** [comment]
Thanks for the pull request :sparkling_heart: 

Just some minor notes

**Comment by typfel** [comment]
Hi, fixed the formatting. Don't know why travis fails though.

**Comment by mikeger** [comment]
@robb I fixed the issue, in 32 bit the expression in the function where resolved as `double` (and `CGFloat` is `float` on 32 bit)

**Comment by robb** [comment]
Can you merge `master`, that should fix matters?

**Comment by robb** [comment]
@mikeger 6c72523 seemed to have undone the code formatting in bbb2623

**Comment by mikeger** [comment]
@robb Indeed, fixed it again

**Comment by robb** [comment]
Thanks guys, glad that RBBAnimation is of use to you :v:

**Pull request merged**